src/commands/operator/index.ts

```diff
@@ -79,6 +91,37 @@ export default class Operator extends OperatorJobAwareCommand {
     this.environment = environment
     this.log('User configurations loaded.')
 
+    if (flags.host === undefined) {
+      this.log(`Using config file for block height track ...`)
+    } else if (this.environment === Environment.localhost || this.environment === Environment.experimental) {
+      this.log(`Skipping API authentication for ${Environment[this.environment]} environment`)
+    } else {
+      this.log(`Using API for block height track ...`)
+      // Create API Service for GraphQL requests
+      try {
+        const logger: Logger = {
+          log: this.log,
+          warn: this.warn,
+          debug: this.debug,
+          error: this.error,
+          jsonEnabled: () => false,
+        }
+        this.apiService = new ApiService(flags.host, logger)
+        await this.apiService.operatorLogin()
+      } catch (error: any) {
+        this.log('Error: Failed to get Operator Token from API')
+        // NOTE: sample of how to do logs when in production mode
+        this.log(JSON.stringify({...error, stack: error.stack}))
+        this.exit()
+      }
+
+      if (this.apiService === undefined) {
+        throw new Error('API service is not defined')
+      }
+
+      this.log(this.apiColor(`API: Successfully authenticated as an operator`))
```

Comment:
This log seems misleading and might concern people that it's authing into a thirdparty service. We should maybe change it to say `Successfully authenticated into API ${hostname}` or something like that